index.js

```diff
@@ -1,57 +1,62 @@
-'use strict';
+'use strict'
 
-// External Modules
-const EventEmitter = require('events').EventEmitter;
-const Logger = require('logdna').Logger;
+const {EventEmitter} = require('events')
+const {createLogger} = require('@logdna/logger')
+const pkg = require('./package.json')
 
 // Constants
 const levels = {
-    10: 'TRACE'
-    , 20: 'DEBUG'
-    , 30: 'INFO'
-    , 40: 'WARN'
-    , 50: 'ERROR'
-    , 60: 'FATAL'
-};
+  10: 'TRACE'
+, 20: 'DEBUG'
+, 30: 'INFO'
+, 40: 'WARN'
+, 50: 'ERROR'
+, 60: 'FATAL'
+}
 
 /*
  *  Support for Bunyan Transport
  */
-class BunyanStream extends EventEmitter {
-    constructor(options) {
-        const pkg = require('./package.json');
-        super();
-        this.logger = new Logger(options.key, Object.assign({}, options, {
-            UserAgent: `${pkg.name}/${pkg.version}`
-        }));
+class LogDNAStream extends EventEmitter {
+  constructor(options) {
+    super()
+    this.logger = createLogger(options.key, {
+      ...options
+    , indexMeta: true
+    , UserAgent: `${pkg.name}/${pkg.version}`
+    })
+
+    this.logger.on('error', (err) => {
+      this.emit('error', err)
+    })
+  }
+
+  write(record) {
+
+    // LogDNA adds their own - lets assume the time delta is trivial
+    // record['timestamp'] = record.time;
```

Comment:
is this comment still relevant, since the `record.timestamp` is now in fact passed along to the logger?

Reply:
It's still valid, but it needs re-located and re-written.  The point it's trying to make is that `bunyan` is providing the timestamp, not LogDNA, but the risk of time drift is low enough to not care.

Reply:
done